### PR TITLE
Add encode_form template filter

### PR DESCRIPTION
**Pull request by stinovlas**

### CODE REVIEW

File: django_forms_encoder/encoder.py [hunk 1]
```diff
@@ -20,41 +20,46 @@ class DjangoFormsEncoder(DjangoJSONEncoder):
...
-
-                form['fields'].append(field_obj)
-            return form
+            return form_to_dict(obj)
 
         return super().default(obj)
+
+
+def form_to_dict(obj):
+    """Convert django form to simple dictionary."""
```
Comment (ziima) [inline django_forms_encoder/encoder.py:29]: Most of the tests for `DjangoFormsEncoder` should be transformed to tests for `form_to_dict`.

File: django_forms_encoder/templatetags/django_forms_encoder.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+from django_forms_encoder.encoder import form_to_dict
+
+register = template.Library()
+
+
+@register.filter
+def encode_form(value):
+    """Convert django form to simple dictionary."""
+    return form_to_dict(value)
```
Comment (ziima) [inline django_forms_encoder/templatetags/django_forms_encoder.py]: `form_to_dict` could be registered directly as a template filter. Or a test should be added.

**Review by ziima — CHANGES_REQUESTED** [review]

File: django_forms_encoder/encoder.py (hunk 1 above)
Comment (stinovlas) [reply django_forms_encoder/encoder.py:29]: Fixed.

File: django_forms_encoder/templatetags/django_forms_encoder.py (hunk 2 above)
Comment (stinovlas) [reply django_forms_encoder/templatetags/django_forms_encoder.py]: Fixed.

**Review by ziima — APPROVED** [review]

**Pull request merged**